readme_collector.py

```diff
@@ -0,0 +1,59 @@
+from coc_collector import *
```

Comment:
i think you should make a shared "ossdi_collector" file which has the ossdi_collector class and then import that here and in the coc_collector